### PR TITLE
Enac conf cleaning

### PR DESCRIPTION
**Pull request by gautierhattenberger**

### CODE REVIEW

**Comment by dewagter** [comment]
make test_enac - ```CobraV2``` not compiling (see #3217)

File: conf/airframes/ENAC/quadrotor/maya_indoor.xml [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    <define name="THRUST_DYNAMICS_FREQ" value="25.64"/>
+    <define name="RC_DEBUG" value="FALSE"/>
+    <define name="SPEED_GAINZ" value="2.5"/>
+  </section>
+
+  <section name="NAV">
+    <define name="ARRIVED_AT_WAYPOINT" value="0.25"/>
+    <define name="NAV_CLIMB_VSPEED" value="0.8"/>
+    <define name="NAV_DESCEND_VSPEED" value="-0.5"/>
+    <define name="NAV_CARROT_DIST" value="18"/>
```
Comment (Fabien-B) [inline conf/airframes/ENAC/quadrotor/maya_indoor.xml:187]: Should we remove this ?

Comment (gautierhattenberger) [reply conf/airframes/ENAC/quadrotor/maya_indoor.xml:187]: not needed but makes no real difference in practice

**Comment by gautierhattenberger** [comment]
No more credit on semaphore, but `test_examples` is now passing

**Review by Fabien-B — APPROVED** [review]

**Pull request merged**